src/ui/pages/Menu/Menu.tsx

```diff
@@ -55,82 +99,81 @@ const Menu = () => {
     );
   };
 
-  const handleItemSelection = (index: number) => {
-    // @TODO - sdisalvo: add some logic for selection
+  const handleCloseSubMenu = () => {
```

Comment:
We don't need this function from your previous commit now. Just pass `setShowSubMenu` to `SubMenu` since within `SubMenu` we call `setShowSubMenu(false)` already.